assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -59,54 +72,66 @@ export const getSortedArrayByKey = (arr, key) => {
     return 0;
   });
   return arr;
-};
+}
 
-export const isDateExpired = (date1, date2) => {
+export const isDateExpired = (date1: number, date2: number) => {
```

Comment:
`export const isDateExpired = (date1: number, date2: number): boolean => {` ?
and same for others below (and above)